mandatory/2-function-creation.js

```diff
@@ -4,7 +4,13 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
```

Comment:
Nice! I think you could simplify the code in a few ways...
1. If you have an `if` statement that looks like this: `if(isLoggedIn === true)` - you can write this instead: `if(isLoggedIn)` - can you think of why that works?
2. Can you think of a way to write this function without the `if` statement? Maybe you can write it with only a `return` statement?